### PR TITLE
kde-frameworks: 5.45 -> 5.46

### PR DESCRIPTION
**Pull request by adisbladis**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @bkchr @ttuegel @peterhoeg

### CODE REVIEW

File: pkgs/development/libraries/kde-frameworks/ki18n.nix [hunk 1]
```diff
@@ -14,9 +14,5 @@ mkDerivation {
   propagatedNativeBuildInputs = [ gettext python ];
   buildInputs = [ qtdeclarative qtscript ];
   patches = [
```
Comment (bkchr) [inline pkgs/development/libraries/kde-frameworks/ki18n.nix]: Why not remove patches completely?

Comment (adisbladis) [reply pkgs/development/libraries/kde-frameworks/ki18n.nix]: Feels silly to have code churn around boilerplate that is likely to come back

Comment (bkchr) [reply pkgs/development/libraries/kde-frameworks/ki18n.nix]: Okay :) It's silly, but much cleaner :D

**Review by ttuegel — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**